### PR TITLE
Brightcom, Brightcom SSP bid adapter: add iiq first party data support

### PR DESCRIPTION
**Pull request by prBigBrother**
## Type of change

- [x] Feature

## Description of change
We have included support for first-party data (as an ID) in the iiq system. We will retrieve and save this data using local storage, but only if the "storageAllowed" setting is enabled

### CODE REVIEW

File: modules/brightcomBidAdapter.js [hunk 1]
```diff
@@ -287,4 +292,72 @@ function _getBidFloor(bid) {
   return null;
 }
 
+function _getFirstPartyData() {
```
Comment (patmmccann) [inline modules/brightcomBidAdapter.js:295]: this appears to be id module functionality; why are you adding to your bidder?

File: modules/brightcomSSPBidAdapter.js [hunk 2]
```diff
@@ -319,4 +324,72 @@ function _getBidFloor(bid) {
   return null;
 }
 
+function _getFirstPartyData() {
```
Comment (patmmccann) [inline modules/brightcomSSPBidAdapter.js:327]: don't copy paste huge code blocks into two adapters, use a library

Comment (prBigBrother) [reply modules/brightcomSSPBidAdapter.js:327]: hi @patmmccann ! I'm not quite sure how libraries are organized in this project. Could you share a few examples with me? I'd greatly appreciate it!

thanks!

Comment (patmmccann) [reply modules/brightcomSSPBidAdapter.js:327]: https://github.com/prebid/Prebid.js/blob/master/libraries/keywords/keywords.js is a code block repeated across multiple adapters. One of our goals to reduce build size is to reduce copy pasted code blocks. 

image

**Pull request closed without merging**